index.html

```diff
@@ -0,0 +1,289 @@
+      <h2 class="softwaredev"> I am software developer</h2>
+      <p class="headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <ul class="socila-link">
+      <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+            alt="Github"></a></li>
+      <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+            src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+      <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+  </section>
+  <section class="desk-first-page">
+    <div class="desk-headline">
+      <h1 class="desk-hey">Hey There I'm Solomon</h1>
+      <h2 class="desk-softwaredev"> I am software developer</h2>
+      <p class="desk-headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <ul class="socila-link">
+      <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+            alt="Github"></a></li>
+      <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+            src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+      <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+  </section>
+  <section class="cards" id="cardds">
+    <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
+    </div>
+    <img class="main-image" src="assets/images/Placeholder.png" alt="main image">
+
+    <div class="card-1">
+      <h2 class="multi-story">Multi-Post Stories</h2>
+      <p class="p21">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="skills">
+        <li class="css-1 button">css</li>
+        <li class="skill button">html</li>
+        <li class="skill button">bootstrap</li>
+        <li class="skill button">Ruby</li>
+      </ul>
+    </div>
```

Comment:
- [ ] Please make sure to remove any HTML code for the projects list instead it should be added via JavaScript dynamically. ✔